### ISSUE TITLE
[Quest] Obelisk of the Sun - Uldum

### ISSUE THREAD

**Issue by BurningXz** [WoWManiaUK/Blackwing-Lair#4353]
https://www.wowhead.com/quest=57990/obelisk-of-the-sun

https://www.youtube.com/watch?v=TiEHM4NofiM

**What is happening:** Npc missing .
when you get the quest (i got him from random drop book , that gives  quest) , you complete him and get 2 more quests from him, but he is not there.

**What should happen:**

**Comment by Echinaceax** [comment]
If im reading this correctly, You got the quest from a book drop and not the npc on the video @BurningXz ?
Can you remember which npc dropped the book and what was the item that dropped called?

**Comment by Xman993** [comment]
@BurningXz , @Echinaceax  this [book](https://www.wowhead.com/item=62768/efficient-excavations)?

**Comment by Echinaceax** [comment]
> @BurningXz , @Echinaceax this [book](https://www.wowhead.com/item=62768/efficient-excavations)?

Not sure @Xman993 but if it is that, then its droping without the prequest being fixed still #4100

**Comment by BurningXz** [comment]
Yes this book.. i think it drops from the mobs nearing the airplanes
